packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.routing.js

```diff
@@ -0,0 +1,85 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers', {
+    url: '/loadbalancers?loadBalancerDeleteSuccess',
+    component: 'octaviaLoadBalancerListing',
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('resources')
+        .then((resources) =>
+          resources.length === 0 ? 'octavia-load-balancer.onboarding' : false,
+        ),
+    resolve: {
+      projectId: /* @ngInject */ ($transition$) =>
+        $transition$.params().projectId,
+      project: /* @ngInject */ ($http, projectId) =>
+        $http.get(`/cloud/project/${projectId}`).then(({ data }) => data),
+      breadcrumb: ($translate) => $translate.instant('octavia_load_balancers'),
+      breadcrumbPrefix: /* @ngInject */ (
+        $injector,
+        $q,
+        coreURLBuilder,
+        project,
+      ) => {
+        if ($injector.has('shellClient')) {
+          return $injector
+            .get('shellClient')
+            .navigation.getURL(
+              'public-cloud',
+              `#/pci/projects/${project.project_id}`,
+            )
+            .then((url) => {
+              return [
+                {
+                  name:
+                    project.status !== 'creating' ? project.description : null,
+                  url,
+                },
+              ];
+            });
+        }
+        return $q.when([
+          {
+            name: project.status !== 'creating' ? project.description : null,
+            url: coreURLBuilder.buildURL(
+              'public-cloud',
+              `#/pci/projects/${project.project_id}`,
+            ),
+          },
+        ]);
+      },
+      goToListingPage: /* @ngInject */ ($state) => () =>
```

Comment:
No need to add goToListingPage here as it already exist on parent state

Reply:
It does not exist on parent state.